report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -1,12 +1,12 @@
-The following chapter contains short introductions to the most important technologies used in the report. 
+The following chapter contains short introductions to the most important technologies used in the project. 
 
 \section*{Glasgow Haskell Compiler} %-- Haskell afsnit istedet??
 Out of the three currently available Haskell compilers available the Glasgow Haskell Compiler (GHC) is used on the backend for interpreting the Haskell exercises and running the test suite. GHC was chosen over the other up-to-date compilers due to it being the standard compiler recommended by the Haskell Organization \cite{Haskell_GHC} as well as it being the compiler used in the Haskell Course.
 
 \section*{Hspec}
 To enable running Haskell tests on the backend we needed a test framework that allowed us to define tests for the exercises. 
 
-Here Hspec was chosen over Hunit since it allows the professor to define the tests in an easy-to-understand domain-specific language(DSL). 
+Here Hspec was chosen over Hunit since it allows the professor to define the tests in an easy-to-understand domain-specific language (DSL). 
```

Comment:
```suggestion
Here Hspec was chosen other test frameworks since it allows the professor to define the tests in an easy-to-understand domain-specific language (DSL). 
```